ui_framework/src/components/header/_header.scss

```diff
@@ -0,0 +1,145 @@
+// Header. Includes breadcrumbs of nav buttons.
+
+$kuiHeaderChildSize: $kuiSizeXXL + $kuiSizeL;
+
+@include component('kuiHeader') {
+  background: $kuiHeaderBackgroundColor;
+  border-bottom: $kuiBorderThin;
+
+  @include child('logo') {
+    position: relative;
+    height: $kuiHeaderChildSize;
+    padding: $kuiSize $kuiSizeXL;
+    display: inline-block;
+    vertical-align: middle;
+
+    &:after {
+      position: absolute;
+      content: "";
+      width: 1px;
+      top: $kuiSize;
+      bottom: 0;
+      right: 0;
+      background: $kuiBorderColor;
+    }
+  }
+
+  @include child('logoIcon') {
+    height: $kuiSizeXL;
+    width: $kuiSizeL;
+  }
+
+  // Truncate long text.
+  @include child('breadcrumbText') {
+    white-space: nowrap;
+    max-width: 150px;
+    display: block;
+    text-overflow: ellipsis;
+    overflow: hidden;
+  }
+
+  @include child('breadcrumb') {
+    display: inline-block;
+    height: $kuiHeaderChildSize;
+    line-height: $kuiHeaderChildSize;
+    padding: 0 $kuiSize;
+    position: relative;
+
+    // Slanted separator. Shows on all but last item.
+    &:not(:last-child):after {
+      position: absolute;
+      top: $kuiSizeL;
+      right: -3px; // Half the transformed width.
+      content: "";
+      width: 1px;
+      height: $kuiSize;
+      transform: rotate(15deg);
+      background: $kuiColorLightGray;
+    }
+
+    @include state('isActive') {
+      color: $kuiColorMediumGray;
+    }
+
+    @include state('isCollapsed') {
+      color: $kuiColorLightGray;
+    }
+  }
+
+  @include child('breadcrumbText') {
+    white-space: nowrap;
+    max-width: 150px;
+    display: block;
+    text-overflow: ellipsis;
+    overflow: hidden;
+  }
+
+  @include child('button') {
+    height: $kuiHeaderChildSize;
+    width: $kuiHeaderChildSize;
+    text-align: center;
+    position: relative;
+
+    &:after {
+      position: absolute;
+      content: "";
+      width: 1px;
+      top: $kuiSize;
+      bottom: 0;
+      left: 0;
+      background: $kuiBorderColor;
+    }
+
+    &:hover {
+      background: $kuiColorLightestGray;
+    }
+
+    &:focus {
+      background: $kuiFocusBackgroundColor;
+    }
+  }
+}
+
+// Laptop
+@include screenMedium {
```

Comment:
Sounds good, but we should probably mention that this deviates from https://sass-guidelin.es/#responsive-web-design-and-breakpoints in `scss_style_guide.md`.